k8s-bench/tasks/create-canary-deployment/cleanup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+NAMESPACE="canary-deployment-ns"
+
+# Delete the namespace
+kubectl delete namespace $NAMESPACE --wait=false
```

Comment:
Lets try to keep these scripts idempotent. For example, in this case, adding command line flag to ignore if the namespace doesn't exist.

We should add hint in our prompt because there might be other things related to idempotency.